scripts/i.in.spotvgt/i.in.spotvgt.py

```diff
@@ -238,22 +238,18 @@ def main():
             gs.fatal(_("An error occurred. Stop."))
 
         # some of the possible values:
-        rules = [
-            r + "\n"
-            for r in [
-                "8 50 50 50",
-                "11 70 70 70",
-                "12 90 90 90",
-                "60 grey",
-                "155 blue",
-                "232 violet",
-                "235 red",
-                "236 brown",
-                "248 orange",
-                "251 yellow",
-                "252 green",
-            ]
-        ]
+        rules = """8 50 50 50
+        11 70 70 70
+        12 90 90 90
+        60 grey
+        155 blue
+        232 violet
+        235 red
+        236 brown
+        248 orange
+        251 yellow
+        252 green
+        """
```

Comment:
Is it possible to verify that the string doesn't have leading whitespace?

If you start by changing to something like the "bad" of this rule: https://docs.astral.sh/ruff/rules/static-join-to-f-string/, so using str.join(), you should end up with ruff making a suggestion for the correct fix.


So, something like removing the list brackets:

```suggestion
        rules = "\n".join([
                "8 50 50 50",
                "11 70 70 70",
                "12 90 90 90",
                "60 grey",
                "155 blue",
                "232 violet",
                "235 red",
                "236 brown",
                "248 orange",
                "251 yellow",
                "252 green",
            ]
        )
```

And then applying the fixes unblocked. Having this inside a list was probably preventing rules to apply at it wouldn't have meant the same.